core/src/main/java/com/flipkart/varadhi/spi/db/MetaStore.java

```diff
@@ -1,5 +1,6 @@
 package com.flipkart.varadhi.spi.db;
 
+import com.flipkart.varadhi.auth.RoleBindingNode;
```

Comment:
+1. separate interface.

Reply:
I tried separating this out but it will lead to introduction of zookeeper dependencies in VerticleDeployer.
Otherwise, we will have to follow the metastore provider pattern for AuthZ too which will involve quite a lot of changes.

Reply:
I tried separating this out but it will lead to introduction of zookeeper dependencies in VerticleDeployer.
Otherwise, we will have to follow the metastore provider pattern for AuthZ too which will involve quite a lot of changes.